### PR TITLE
Fixed README.md

### PR DESCRIPTION
**Pull request by colinksmith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,3 @@
 Repo for collaboration on a social app that will be used to recommend tv and streaming service shows to like-minded individuals. 
 
 Start your server with "npm run dev", terminal should tell you what localhost you need!
-
-testing
```
Comment (DavidRod1865) [inline README.md:6]: **Why are you deleting two lines?**

**Review by DavidRod1865 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (colinksmith) [reply README.md:6]: They are superflous

Comment (DavidRod1865) [reply README.md:6]: I don't know... seems fishy.

**Review by DavidRod1865 — APPROVED** [review]
lgtm

**Pull request merged**